src/components/InstallButton.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Dropdown, Link } from '@newrelic/gatsby-theme-newrelic';
+import { css } from '@emotion/react';
+
+const sampleItems = new Array(10).fill().map((_, i) => i + 1);
+
+const createMenuItems = (children, items) => {
+  const menuItems = [];
+  for (const item of items) {
+    menuItems.push(<Dropdown.MenuItem>{item}</Dropdown.MenuItem>);
+  }
+  return menuItems;
+};
+
+const createInstallLink = () => {
+  return `https://one.newrelic.com/launcher/nr1-core.explorer`;
+};
+
+const InstallButton = ({ children, title, ...props }) => {
+  return (
+    <div
+      css={css`
+        display: flex;
+        flex-direction: row;
+        float: right;
```

Comment:
`float` is traditionally used on images or other content that you need the rest of the pages content to flow around. So I'm a little wary about using this on a button. What is it accomplishing here? I'm sure there's another way to get the desired behavior.

Reply:
@aswanson-nr ooh yes good call I was just using that for display purposes for screenshots, probably don't need to include any sort of styling like that in the component itself do we? Leave that in the layout?

Reply:
Leaving it to the parent component/layout sounds like the way to go! That way we can use it in multiple contexts, if needed.